examples/Design-class-DesignGrouped.R

```diff
@@ -57,13 +57,17 @@ design <- DesignGrouped(
   mono = one_arm
 )
 
-# Alternative options: Here e.g. use both mono and combo from the start,
-# but allow different dose levels for the cohorts. Stop mono arm too, when combo
-# arm is stopped.
+# Alternative options: Here e.g.
+# - use both mono in first cohort and afterwards have mono and combo in parallel,
+# - but allow different dose levels for the cohorts.
+# - Stop mono arm too, when combo arm is stopped.
+# - Start parallel though, i.e. for the second cohort where mono and combo start together,
+#   use the same dose
```

Comment:
Again, the language here is awkward. So awkward, I'm not sure what you are trying to say!  "start in parallel" suggests that both mono and combo are used in the first cohort, so "for the second cohort where mono and combo start together..." is contradictory.